app/src/main/java/com/github/swent/echo/data/model/Event.kt

```diff
@@ -7,9 +7,8 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class Event(
     val eventId: String,
-    val creatorId: String,
-    val organizerId: String,
-    val organizerName: String,
+    val creator: UserProfile,
+    val organizer: Association,
```

Comment:
We should make this nullable as not every event will be hosted by an association...